sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DataSourceV2Strategy.scala

```diff
@@ -329,10 +329,17 @@ class DataSourceV2Strategy(session: SparkSession) extends Strategy with Predicat
       }
       DescribeTableExec(output, r.table, isExtended) :: Nil
 
-    case DescribeColumn(_: ResolvedTable, column, isExtended, output) =>
+    case DescribeColumn(r: ResolvedTable, column, isExtended, output) =>
       column match {
         case c: Attribute =>
-          DescribeColumnExec(output, c, isExtended) :: Nil
+          val colStats =
+            r.table.asReadable.newScanBuilder(CaseInsensitiveStringMap.empty()).build() match {
```

Comment:
this may not be very cheap. Shall we only do it if `isExtended` is true?

Reply:
and what if the table is not readable (e.g. write-only)? We should not fail but show no column stats.

Reply:
BTW, I think it's cleaner to pass v2 `Table` to `DescribeColumnExec` and move this code block to `DescribeColumnExec`

Reply:
@cloud-fan Thanks for your comments. I will have a follow-up to fix this and also move the test to parent suite.